dpnp/dpnp_iface_arraycreation.py

```diff
@@ -2183,7 +2188,7 @@ def geomspace(
 
     """
 
-    return dpnp_geomspace(
+    res = dpnp_geomspace(
```

Comment:
Input args have dpnp.ndarray type, but output arg has usm_ndarray type. Looks unclear.
Can we move this code inside dpnp_geomspace? Or use usm_ndarray as input type for dpnp_geomspace function?
```
dpnp.synchronize_array_data(res)
res = dpnp_array._create_from_usm_ndarray(res)
```

Reply:
I will update to return dpnp array.